### PR TITLE
[TS-LATAM | Master ] LPS-132648  using style compatible with outlook 

### PR DESCRIPTION
**Pull request by rijema**
Hi team! Here is the link to the [LPP](https://issues.liferay.com/browse/LPP-41106) & [LPS](https://issues.liferay.com/browse/LPS-132648).

I'm available in case of any doubts.

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.ftl [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		.field-value {
 			font-size: 16px;
 			margin: 0;
-			white-space: pre-line;
+			white-space: pre;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.ftl:30]: @rijema, do we really need this CSS property or the tag `<pre>` is enough to attend our needs?

Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.ftl:30]: Hi @natocesarrego , I also tested only with the tag pre and yes, it solves both use cases (the one whose I solved before using the CSS attribute and this one specifically to the Outlook). I was just double checking with the CSS change. Do you want me to update this PR with only the tag change?

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.ftl:30]: Yes, please, @rijema. :slightly_smiling_face:

**Comment by rijema** [comment]
https://github.com/liferay-forms/liferay-portal/pull/695

**Pull request closed without merging**